src/documentation.ts

```diff
@@ -115,6 +126,11 @@ export class Documentation extends OpenApiBuilder {
       _method,
     ) => {
       const method = _method as Method;
+      const operationId = this.ensureUniqOperationId(
+        path,
+        method,
+        endpoint.getOperationId(),
+      );
```

Comment:
consider moving it below and placing before the `const operation`, just for keeping those things closer to each other.